src/mixed-line-bar-chart/__integ__/mixed-line-bar-chart.test.ts

```diff
@@ -336,6 +336,35 @@ describe('Details popover', () => {
     })
   );
 
+  test(
+    'can be hidden after hover by pressing Escape',
+    setupTest('#/light/mixed-line-bar-chart/test', async page => {
+      // Hover over first group
+      await page.hoverElement(chartWrapper.findBarGroups().get(1).toSelector());
+      await expect(page.getText(popoverHeaderSelector())).resolves.toContain('Potatoes');
+
+      // Pressing escape should hide the popover
+      await page.keys(['Escape']);
+      await expect(page.isDisplayed(popoverDismissSelector())).resolves.toBe(false);
+      await expect(page.isDisplayed(popoverHeaderSelector())).resolves.toBe(false);
+    })
+  );
+
+  test(
+    'can be hidden after keyboard navigation by pressing Escape',
+    setupTest('#/light/bar-chart/test', async page => {
```

Comment:
Is it intentional that this test navigates to `'#/light/bar-chart/test'` even though the test file name is `mixed-line-bar-char-test.ts`? If we are testing the Bar Chart, would it make more sense to move this test to the existing file `bar-chart/__tests__/bar-chart.test.ts`?

Reply:
good point. I understand that the component is common for both chart variants, but still a bit confusing

Reply:
ok, now it makes more sense :)